README.md

```diff
@@ -37,7 +37,11 @@ install `aspire` safely in that environment.
 
 If you are unfamiliar with `conda`, the
 [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
-distribution is recommended.
+distribution for `x86_64` is recommended.  For Apple silicon to use
```

Comment:
This sentence sounds just a little funny.

Reply:
Sure, I will update it since you guys didn't like it so much.